### DISCUSSION TITLE
Unblock devtools + Flags

### DISCUSSION THREAD

**Discussion by Luigi-Master** [3kh0/ext-remover#1403] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have chrome flags enabled and I want to try this chrome flag exploit but 1. is this patched?, and 2. how do I use a vi?

**Comment by schoolexploitkid** [comment]
what exploit is it

**Comment by akabutnicer** [comment]
i love when you elaborated the exploit you were going to use

**Reply by awsome-webdev** [reply]
lol

**Comment by Luigi-Master** [comment]
my bad for not being specific, here is the link, https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Unblocked%20Dev%20Tools%20%2B%20Flags

**Reply by Luigi-Master** [reply]
and go to the section unblock dev tools + flags

**Reply by akabutnicer** [reply, marked as answer]
i want to confirm that we both interpreted the part that says, 'you will need some knowledge of bash'. not sure if we did, but if you do not know bash as it suggests, you should be attempting some other exploit.

**Reply by Luigi-Master** [reply]
ok
